src/io/result_io.jl

```diff
@@ -369,13 +369,13 @@ function setup_result_store!(domain::Domain, scen_spec::DataFrame)::Tuple
     mkdir(joinpath(log_location, SPATIAL_DATA))
     geo_fn = joinpath(log_location, SPATIAL_DATA, basename(attrs[:name]) * ".gpkg")
     try
-        GDF.write(geo_fn, domain.site_data; driver="geojson")
+        GDF.write(geo_fn, domain.site_data; driver="geopackage")
```

Comment:
```suggestion
        GDF.write(geo_fn, domain.site_data; driver="GPKG")
```
Is `GPKG` the intended driver?

using `geopackage` causes a GDAL error.
```
ERROR: GDALError (CE_Failure, code 10):
        Pointer 'hDriver' is NULL in 'GDALCreate'.
```

Reply:
Ah, I thought I had changed that!